lasagne/layers/shape.py

```diff
@@ -98,8 +95,8 @@ def get_output_shape_for(self, input_shape, *args, **kwargs):
                                      (o[0], len(input_shape)))
                 output_shape[dim] = input_shape[o[0]]
                 masked_output_shape[dim] = input_shape[o[0]]
-                if ((input_shape[o[0]] is None)
-                    and (masked_input_shape[o[0]] is None)):
+                if (input_shape[o[0]] is None) \
+                   and (masked_input_shape[o[0]] is None):
```

Comment:
> Yes, two indents gives the same error, `E129 visually indented line with same indent as next logical line`.

Really? Shouldn't it say `indented too much for visual indent` (whatever the wording of that is)?

> We can double indent when using line continuation, would you prefer that?

I'd prefer that, but I don't know what the majority thinks. @benanne, @dnouri: What do you prefer for long if clauses? There's an overview of alternatives here, for example: http://stackoverflow.com/a/181557